Documentation/csi_block_storage_kc_pdfs.md

```diff
@@ -0,0 +1,42 @@
+# Publications and related information
+
+The following Portable Document Format \(PDF\) publications and related online information sources are available for the IBM® block storage CSI driver.
+
+To view a PDF file, you need Adobe™ Reader. You can download it at no charge from the [Adobe website](http://get.adobe.com/reader/).
+
+## Publications
+
+|IBM block storage CSI driver publication|Description|PDF download link|
+|----------------------------------------|-----------|-----------------|
+|*Release notes*|This publication describes requirements, compatibility, change log, and known issues information for version 1.6.0.<br />First Edition \(June 2021\) for version 1.6.0.|![PDF icon](PDF_icon.jpg) [Download](PDFs/IBM_block_storage_CSI_driver_1.6.0_RN.pdf)|
+|*User guide*|This publication describes how to prepare for, install, configure, and use IBM block storage CSI driver.<br />Publication number: SC27-9590-07.|![PDF icon](PDF_icon.jpg)[Download](PDFs/IBM_block_storage_CSI_driver_1.6.0_UG.pdf)|
+
+{% reusables.doc-resources %}
```

Comment:
I [don't think it worked](https://github.com/IBM/ibm-block-csi-driver/blob/0e520e127e74434580e5d39f8095ef948f3f78f7/Documentation/csi_block_storage_kc_pdfs.md#publications)

Reply:
the information is there and showing properly. it's the pdf links, i think that are you saying are not showing up.
this i know. i am trying to figure out if I can still create the PDFs and how to properly do so with this new format.

Reply:
resolving conversation. this is now listed differently from a different source